### DISCUSSION TITLE
posts/ai-poem-art/

### DISCUSSION THREAD

**Discussion by giscus** [brant-ruan/fernweh-comments#13] [Announcements]
# posts/ai-poem-art/

计算诗歌——ChatGPT凭诗作画的尝试

https://blog.wohin.me/posts/ai-poem-art/

**Comment by Drag0nf1y** [comment]
博主，可以创个云安全、系统安全的telegram交流群吗，可以任意分享博文和技术，以及提问什么的。”在线社区“

**Reply by brant-ruan** [reply]
谢谢你的提议，不过我暂时没有这个打算。

**Reply by Drag0nf1y** [reply]
那好吧，谢谢您的回复
